compose.yml

```diff
@@ -20,33 +21,44 @@ x-redash-environment: &redash-environment
   REDASH_ENFORCE_CSRF: "true"
   REDASH_GUNICORN_TIMEOUT: 60
   # Set secret keys in the .env file
+=======
+version: "3.8"
+>>>>>>> reuse built frontend in ci, merge compose files:compose.yml
```

Comment:
This looks like a problem left over from a merge conflict.

Reply:
@justinclift fixed it

Reply:
@justinclift tested this PR in my fork
https://github.com/AndrewChubatiuk/redash/actions/runs/8223358936
https://github.com/AndrewChubatiuk/redash/actions/runs/8223482138